nxc/protocols/ssh.py

```diff
@@ -15,13 +12,28 @@
     SSHException,
 )
 
-
 class ssh(connection):
     def __init__(self, args, db, host):
         self.protocol = "SSH"
-        self.remote_version = None
-        self.server_os = None
+        self.remote_version = ""
+        self.server_os_platform = "Linux"
+        self.user_principal = "root"
         super().__init__(args, db, host)
+    
+    def proto_flow(self):
```

Comment:
The Debug statement from the original connection.py would be nice to have here

Reply:
Fixed